### PR TITLE
added cron to calculate common score

### LINKED ISSUE #1300: Sort commons in Explore page [S12-02]
Sorting common in explore by the following SCORE formula calculation of: 

- Number of members*100,
- balance in shekels *20,
- Raised in shekels *5,
- Number of proposals *30,
- Number of discussions *10,
- Number of posts *1
- Last updated in days -*100
- Time since creation in days *2

Highest score first

This feature will be instead the “featured” mechanism. Hence, all public commons will be shown in the explore ordered by their scoring. 

Private commons will be shown in explore only for members (for non members they could accessed by direct link)

### PR DESCRIPTION
**Pull request by noamso1**

### CODE REVIEW

**Comment by noamso1** [comment]
@ariel-dao do we also need to
1. Create an end-point for fetching the commons sorted by score
2. Extend an existing end-point for fetching the commons, that will sort by score
3. Do nothing - FE will just fetch directly from db and will ask to sort by score

**Comment by ariel-dao** [comment]
@noamso1 the 3rd option

**Comment by noamso1** [comment]
@ariel-dao so please approve and merge ...

File: functions/src/crons/commonScores.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import * as functions from 'firebase-functions';
+import { firestore } from 'firebase-admin';
+import { commonsDb } from '../commons/db'
+
+export const commonScores = functions.pubsub
+  .schedule('29 */4 * * *') // every 4 hours at the 29th minute
+  .onRun(async () => {
+
+    let commons = await commonsDb.getMany( {active: true })
```
Comment (ariel-dao) [inline functions/src/crons/commonScores.ts]: const

**Review by ariel-dao — CHANGES_REQUESTED** [review]
change please

**Review by ariel-dao — APPROVED** [review]

**Pull request merged**